### DISCUSSION TITLE
Actuator end points access behind firewall

### DISCUSSION THREAD

**Discussion by THENNARASU** [codecentric/spring-boot-admin#3754] [Ideas]
In a case where the services are ONLY accessible through api gateway, the actuator url's wont work.

It would be ideal to access the endpoint URL's within the spring boot admin on the server side and present the content to the client.

**Comment by hzpz** [comment]
To understand your use case, we need a little more information about what you're trying to achieve.

The Spring Boot Admin server needs access to the actuator endpoints of the services it should monitor. The Spring Boot Admin web interface however, needs access the the Spring Boot Admin server only. Every information from the actuator endpoints is routed through the server to the web interface. So, if both the services and the Spring Boot Admin server are behind the API gateway, you would only need to make sure, that the Spring Boot Admin server is accessible through the gateway.
